__tests__/pages/index.spec.tsx

```diff
@@ -1,9 +1,9 @@
-import * as React from 'react';
-import * as TestRenderer from 'react-test-renderer';
-import Home from '../../pages/index';
+import * as React from "react";
+import * as TestRenderer from "react-test-renderer";
+import Home from "../../pages/index";
 
-describe('Home test', () => {
-  it('tests home page render correctly', () => {
+describe("Home test", () => {
+  it("tests home page render correctly", () => {
```

Comment:
These are effected by Prettier, right? Supposed not to change in this PR.

Reply:
Yes, I thought I am supposed to format before the merge, no?

Reply:
If it is, it is supposed to be formatted in Prettier's PR, right?

Every PR should have a single responsibility. In order to format this file, we should create another PR that has responsibility to format it.

Reply:
I understand! thank you.